authors/apps/profiles/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+from django.db.models.signals import post_save, pre_save
+from django.utils.text import slugify
+from ..authentication.models import User,UserManager
+from cloudinary.models import CloudinaryField
+
+
+class UserProfile(models.Model):
+    """
+    This class contains  models relating to user profile. 
+    It has a one to one relationship with User model class.
+    An intance of this class is created during user registration by calling 
+    the create profile in the post_save function
+    """
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    # A char field to hold bio data
+    bio = models.TextField(max_length=200, default='')
+
+    # A choice field to hold multiple choice 
+    gender_choices = (('M','MALE'),('F','FEMALE'))
+    gender = models.CharField(max_length=10, choices=gender_choices, default='M')
+
+    # cloudinary field
+    profile_image = CloudinaryField('image', null=True, default='', blank=True)
+
+    # A char field for the First name
+    first_name = models.CharField(max_length=100, default='')
+
+    # A char field for the user Last name
+    last_name = models.CharField(max_length=100, default='')
```

Comment:
Let's have a timestamp field, probably `updated_at` so we keep record of when provide was edited. We can always get `created_at` from User.